src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -75,29 +84,227 @@ public DarCollectionService(DarCollectionDAO darCollectionDAO, DarCollectionServ
    * @param userRole The role the user is making the request as
    * @return List of DarCollectionSummary objects
    */
-  public List<Object> getSummariesForRoleName(User user, String userRole) {
-    switch (userRole) {
-      case Resource.ADMIN:
-        // TODO: Find all
+  private void updateStatusCount(Map<String, Integer> statusCount, String status) {
+    Integer count = statusCount.get(status);
```

Comment:
This can throw an NPE if the key is null. Try `containsKey()` instead

Reply:
True, but `Map` doesn't support null keys, so this method is relying on callers to pass in the correct implementation.

Reply:
Ah, which means that I'll have to check on null to keep Map methods from erroring out.